src/runtime/virtcontainers/acrn.go

```diff
@@ -214,7 +214,7 @@ func (a *Acrn) trace(parent context.Context, name string) (otelTrace.Span, conte
 	}
 
 	tracer := otel.Tracer("kata")
-	ctx, span := tracer.Start(parent, name, otelTrace.WithAttributes(label.String("source", "runtime"), label.String("package", "virtcontainers"), label.String("subsystem", "hypervisor"), label.String("type", "acrn")))
+	ctx, span := tracer.Start(parent, name, otelTrace.WithAttributes(label.String("source", "runtime"), label.String("package", "virtcontainers"), label.String("subsystem", "hypervisor"), label.String("type", "acrn"), label.String("id", a.id)))
 
```

Comment:
It's better to use `sandbox_id` instead `id` ?

Reply:
@liubin I changed it but this I'm not completely sure about. In fc there is a comment that it normally maps to the sandbox ID, and for qemu it does. Do you know if this the case for acrn and clh?

Reply:
@cmaf It seems that only firecracker has that limitation, the id should be shorter than 32 bytes. Acrn/clh are all using the sandbox ID. So using `sandbox_id` may not lead to any misunderstanding.